Bottom: ba334056e58432033c2b6ed101e105a6118e6a30
Top:    33c7aadf8e0e42df46840fbf09d4dce9b40ee49a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-13 15:36:13 -0500

Refresh of asset-host

---
diff --git a/roles/buildbot-worker-ptxdist/templates/netrc.j2 b/roles/buildbot-worker-ptxdist/templates/netrc.j2
new file mode 100644
index 00000000..e81d1062
--- /dev/null
+++ b/roles/buildbot-worker-ptxdist/templates/netrc.j2
@@ -0,0 +1,3 @@
+machine {{ asset_host | urlsplit('hostname') }}
+login {{ asset_user }}
+password {{ asset_password }}
diff --git a/roles/buildbot-worker-ptxdist/templates/ptxdistrc.j2 b/roles/buildbot-worker-ptxdist/templates/ptxdistrc.j2
new file mode 100644
index 00000000..4d2b48e9
--- /dev/null
+++ b/roles/buildbot-worker-ptxdist/templates/ptxdistrc.j2
@@ -0,0 +1,76 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# PTXdist 2012.09.1-00373-gee9622f
+#
+
+#
+# PTXDIST Setup Menu
+#
+
+#
+# User
+#
+PTXCONF_SETUP_USER_NAME="NovaTech Buildbot"
+PTXCONF_SETUP_USER_EMAIL="buildbot@novatechweb.com"
+
+#
+# Proxies
+#
+PTXCONF_SETUP_FTP_PROXY=""
+PTXCONF_SETUP_HTTP_PROXY=""
+PTXCONF_SETUP_HTTPS_PROXY=""
+PTXCONF_SETUP_NO_PROXY=""
+
+#
+# Project Searchpath
+#
+PTXCONF_SETUP_PROJECTPATH="${PTXDIST_TOPDIR}/projects"
+
+#
+# Source Directories
+#
+PTXCONF_SETUP_SRCDIR="{{cache_path}}/downloads"
+
+#
+# Source Download
+#
+# PTXCONF_SETUP_NO_DOWNLOAD is not set
+# PTXCONF_SETUP_PTXMIRROR_ONLY is not set
+PTXCONF_SETUP_PTXMIRROR="{{asset_host}}/orphans"
+PTXCONF_SETUP_DEBMIRROR="{{asset_host}}/debian"
+PTXCONF_SETUP_SFMIRROR="{{asset_host}}/sourceforge"
+PTXCONF_SETUP_GNUMIRROR="{{asset_host}}/gnu"
+PTXCONF_SETUP_XORGMIRROR="{{asset_host}}/xorg"
+PTXCONF_SETUP_KERNELMIRROR="{{asset_host}}/kernel"
+PTXCONF_SETUP_PYPIMIRROR="{{asset_host}}/pypi"
+# PTXCONF_SETUP_CHECK_ALWAYS is not set
+PTXCONF_SETUP_CHECK_NOTEMPTY=y
+# PTXCONF_SETUP_CHECK_NEVER is not set
+PTXCONF_SETUP_CHECK="notempty"
+
+#
+# IPKG Repository
+#
+PTXCONF_SETUP_IPKG_REPOSITORY="{{cache_path}}/ipkg-repository"
+
+#
+# Java SDK
+#
+PTXCONF_SETUP_JAVA_SDK="/usr"
+
+#
+# Developer Options
+#
+# PTXCONF_SETUP_DISABLE_LOCAL_CHECK is not set
+PTXCONF_SETUP_ENV_WHITELIST=""
+# PTXCONF_SETUP_COMMON_CACHE is not set
+# PTXCONF_SETUP_GEN_DEP_TREE is not set
+# PTXCONF_SETUP_CHECK_EXIT_ON_ERROR is not set
+PTXCONF_SETUP_CCACHE=y
+PTXCONF_SETUP_PATCHIN_GIT=y
+# PTXCONF_SETUP_NFS_REL_SYMLINK is not set
+# PTXCONF_SETUP_DIRECT_CLEAN is not set
+PTXCONF_SETUP_HOST_CPP="cpp"
+PTXCONF_SETUP_HOST_CC="gcc"
+PTXCONF_SETUP_HOST_CXX="g++"
+PTXCONF_SETUP_HOST_MAKE="make"
